src/DIRAC/TransformationSystem/Client/TransformationStatus.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module contains constants and lists for the possible transformation states.
+"""
+
+#:
+NEW = "New"
+#:
+IDLE = "Idle"
+#:
+ACTIVE = "Active"
+#:
+FLUSH = "Flush"
+#: Status for transformations that gets "derived"
```

Comment:
```suggestion
#: Status for transformations that are being "derived" from
```
To make clear that this status belongs to the transformation that another transformation is currently being derived from. (I think that is what was meant).

(`gets` as written before would refer to Status, but one doesn't derive from the status, does one? So it should be `get` but that seems to imply the the derived Transformation has the status Completing...)